CONTRIBUTING.md

```diff
@@ -185,6 +185,27 @@ It is a good practice to keep your branch updated by rebasing your branch to mai
 * If you feel like your PR is waiting too long for a review, feel free to ping the [`#thanos-dev`](https://slack.cncf.io/) channel on our slack for a review!
 * If you are a new contributor with no write access, you can tag in the respective maintainer for the changes, but be patient enough for the reviews. *Remember, good things take time :)*
 
+### Pull Request Title Format
+
+To ensure a consistent and meaningful naming convention for pull requests (PRs), please follow these guidelines:
+
+#### PR Title Format
+
+PR titles should follow this pattern:
+- `prefix: brief description`
+
+Where `prefix` can be:
+- `query`: If your PR is related to queries.
+- `something`: A general prefix indicating the component or part of the system affected by your PR. This should be specific to the area of the codebase you are changing.
+
+#### Examples
+- `query: Optimize search query performance`
+- `query: Add new filters to the query module`
+- `store: Improve data storage efficiency`
+- `api: Update API endpoint for user data`
```

Comment:
What about multi-component PRs like https://github.com/thanos-io/thanos/pull/7406?

What about a PR like this one itself?

Reply:
@douglascamata What about this [docs/components/tools.md](https://github.com/thanos-io/thanos/pull/7470/files/4d41d7036df1de28dc40ca5553525102da7e36e9#diff-89840bcebb29ac15c25fad738d4d95c327074e0559da78e4851b696d9533c267) is this related to my work? 

For PRs involving multiple components, maybe we can use syntax like this -> `query,store: Enhance data retrieval efficiency`. What you say?

Reply:
Seems okay to me. But mind that I'm not a maintainer, so my review is not conclusive. A maintainer will still have to review this.